llvm/test/Analysis/CostModel/AMDGPU/load_i8.ll

```diff
@@ -0,0 +1,466 @@
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py
+; RUN: opt -passes="print<cost-model>" 2>&1 -disable-output -mtriple=amdgcn-unknown-amdhsa -mcpu=gfx90a < %s | FileCheck %s --check-prefixes=GFX90A
+
+define void @loads_align4(i32 %arg) {
+  ; Scalars
+; GFX90A-LABEL: 'loads_align4'
+; GFX90A-NEXT:  Cost Model: Found an estimated cost of 1 for instruction: %1 = load i8, ptr poison, align 4
```

Comment:
Can you add tests for different types too (e.g. i1 and i16) and different addressspaces.